### PR TITLE
Draft of readme and ipynb content updates

### PR DESCRIPTION
**Pull request by HeidiSteen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,82 @@
...
+For the purpose of this example, we won't use the *OnResults* event for calculations.
 
-Detailed steps can be found in the following Jupyter [notebook](notebooks/app_insights_sdk_example.ipynb)
-  
+Detailed steps can be found in the following Jupyter Notebook [app_insights_sdk_example.ipynb](notebooks/app_insights_sdk_example.ipynb).
   
-## Deploying the Dashboard
+## Deploy to Azure
+
+As a first step, deploy the dashboard to your Application Insights resource. You'll select the blue button below, which prepares a [predefined ARM template](/src/arm/dashboard.template.json), and then fill in some properties. The ARM template creates the visualizations and provides the Kusto queries that returns simulated data.
```
Comment (melmatlis) [inline README.md:47]:
```suggestion
As a first step, deploy the dashboard to your Application Insights resource. You'll select the blue button below, which prepares a [predefined ARM template](/src/arm/dashboard.template.json), and then fill in some properties. The ARM template creates the visualizations and provides the Kusto queries that explore the simulated data.
```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,82 @@
...
+
+1. Create a `.env` file and paste in the following settings. Substitute a valid instrumentation key for the placeholder string. You can get the instrumentation key from the Overview page for Application Insights in the Azure portal.
 
-This code is a collaborative effort of CSE & DAI&SL<br>
-Huge credit to [@dupuyjs](https://github.com/dupuyjs) for setting up the ARM template and expertise
+   ```ini
+    API_LOGGING_LEVEL = "DEBUG"
+   LOGGER_NAME = "test_logger"
+   APP_INSIGHTS_KEY = "<your-application-insights-instrumentation-key>"
+   ```
```
Comment (melmatlis) [inline README.md:81]:
```suggestion
## Contribution
This code is a collaborative effort of CSE & DAI&SL<br>
Huge credit to [@dupuyjs](https://github.com/dupuyjs) for setting up the ARM template and expertise

```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,82 @@
...
+This code is a collaborative effort of CSE & DAI &SL.<br>
+Huge credit to [@dupuyjs](https://github.com/dupuyjs) for setting up the ARM template and expertise.
+
+## Set up your environment
+
+1. Start Visual Studio Code and open the root folder of the sample.
+
+1. Create a `.env` file and paste in the following settings. Substitute a valid instrumentation key for the placeholder string. You can get the instrumentation key from the Overview page for Application Insights in the Azure portal.
 
-This code is a collaborative effort of CSE & DAI&SL<br>
```
Comment (melmatlis) [inline README.md:45]: This work is and excerpt from a client project, I would like to keep the contribution to others in the READ.ME file.
I hope it's OK, I moved to the section prior to licensing

**Review by melmatlis — CHANGES_REQUESTED** [review]
Thank you @HeidiSteen  for this amazing PR.
The content is so much clearer than it was, really appreciate the time and effort you took into improving our work

File: notebooks/app_insights_sdk_example.ipynb [hunk 4]
```diff
@@ -27,17 +28,46 @@
...
+    "  ```\n",
+    "  API_LOGGING_LEVEL = \"DEBUG\"\n",
+    "  LOGGER_NAME = \"test_logger\"\n",
+    "  APP_INSIGHTS_KEY = \"<your-app-insights-instrumentation-key>\"\n",
+    "  ```\n",
+    "\n",
+    "* Install the packages listed in the requirements TXT file. Packages include [opencensus-ext-azure](https://pypi.org/project/opencensus-ext-azure/), [python-dotenv](https://pypi.org/project/python-dotenv/), and [Faker](https://pypi.org/project/Faker/).\n",
+    "\n",
+    "  ```\n",
+    "  %pip install -r ../requirements.txt\n",
```
Comment (melmatlis) [inline notebooks/app_insights_sdk_example.ipynb:54]:
```suggestion
    "  !pip install -r ../requirements.txt\n",
```

Comment (HeidiSteen) [reply notebooks/app_insights_sdk_example.ipynb]: I changed this back to how you had it before.  VS Code suggested the alternate syntax, but you know this space best.

**Review by HeidiSteen — COMMENTED** [review]
@melmatlis I think everything is addressed. I moved the credit up to the intro in the readme (line 15), and spelled out the organization name.  Unless you spot something else, I think the PR is ready.

**Pull request merged**